### PR TITLE
Blog post about RGSoC day 1 and campaign wrap up

### PR DESCRIPTION
**Pull request by anasofiapinho**
Publish a blog post about RGSoC 2016 kick-off day and campaign wrap up.

### CODE REVIEW

**Comment by anasofiapinho** [comment]
@mkalininait or @anikalindtner do you mind taking a look?

File: blog/_posts/2016-07-01-RGSoC-2016-day-1.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+![RGSoC board transforms!](/img/blog/2016/2016-07-01-RGSoC-2016-day-one-stop-motion.gif)<font color="grey"><small><i>The RGSoC thank_you board transforming throughout the past few months. (Image: Ana Sofia Pinho)</i></small></font>
+
+Until now! Those red squares were saved for our RGSoC 2016 teams: students, coaches, mentors and supervisors. More than anyone else, these teams will be dedicating their time and energy to this project, sharing their knowledge and giving back to the community. So, now that we are onboarding all the teams we can finally add them to our board! Whohoo! \o/
+
+![The teams of RGSoC 2016 on the thank_you board.](/img/blog/2016/2016-07-01-RGSoC-2016-day-one-teams.jpg)<font color="grey"><small><i>The teams of RGSoC 2016 on the thank_you board. (Image: Ana Sofia Pinho)</i></small></font>
```
Comment (anikalindtner) [inline blog/_posts/2016-07-01-RGSoC-2016-day-1.md]: this image is giving a 404 error

File: blog/_posts/2016-07-01-RGSoC-2016-day-1.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+created_at: Fri Jul 01 2016
+permalink: blog/2016-07-01-RGSoC-2016-day-one
+current: blog
+author: Ana Sofia
+twitter: RailsGirlsSoC
+categories:
+- news
+---
+
+![Our thank_you board is almost complete!](/img/blog/2016/2016-07-01-RGSoC-2016-day-one-board.png)<font color="grey"><small><i>Our thank_you board is finished! Have you seen the new red squares on top? :) (Image: Ana Sofia Pinho)</i></small></font>
```
Comment (anikalindtner) [inline blog/_posts/2016-07-01-RGSoC-2016-day-1.md]: "Have you seen" -> "Do you see"

File: blog/_posts/2016-07-01-RGSoC-2016-day-1.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+The countdown to Rails Girls Summer of Code is over! \o/ Can you believe it!? July 1st is finally here! For the next 3 months, 40 women from 11 countries will be dedicating their summer — or winter, depending on the hemisphere they live in, — to work on Open Source projects and take their software development skills to another level. This summer is going to be one of the best summers of their lives and you know what? You made this happen!
```
Comment (anikalindtner) [inline blog/_posts/2016-07-01-RGSoC-2016-day-1.md]: I'd strike the \o/

File: blog/_posts/2016-07-01-RGSoC-2016-day-1.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+As you might recall, a couple of months ago, [we created our thank_you board](http://railsgirlssummerofcode.org/blog/2016-04-22-thank-you-lets-diversify-tech) and gave each individual donor a square with their name. We applied the same principle to each sponsor and attributed them a group of cards depending on the sponsorship package. We hope you liked this tiny and heartfelt gesture. <3 As time went by, the board transformed — or shall we say became more diverse — and along the way you might have wondered why we never added a square to the top three rows...
+
+![RGSoC board transforms!](/img/blog/2016/2016-07-01-RGSoC-2016-day-one-stop-motion.gif)<font color="grey"><small><i>The RGSoC thank_you board transforming throughout the past few months. (Image: Ana Sofia Pinho)</i></small></font>
+
+Until now! Those red squares were saved for our RGSoC 2016 teams: students, coaches, mentors and supervisors. More than anyone else, these teams will be dedicating their time and energy to this project, sharing their knowledge and giving back to the community. So, now that we are onboarding all the teams we can finally add them to our board! Whohoo! \o/
```
Comment (anikalindtner) [inline blog/_posts/2016-07-01-RGSoC-2016-day-1.md]: I'd strike the \o/

File: blog/_posts/2016-07-01-RGSoC-2016-day-1.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+![The teams of RGSoC 2016 on the thank_you board.](/img/blog/2016/2016-07-01-RGSoC-2016-day-one-teams.jpg)<font color="grey"><small><i>The teams of RGSoC 2016 on the thank_you board. (Image: Ana Sofia Pinho)</i></small></font>
+
+#### <span class="color-red">It’s time to close our fundraising campaign</span>
+As the Summer of Code begins, the fundraising campaign for RGSoC 2016 ends. And just like we did on previous years, we are now opening the fundraising campaign for 2017. This means that if you want to contribute to our program, [you can still do it](http://railsgirlssummerofcode.org/campaign/), but all the money we receive from now on will be used on the RGSoC 2017’s edition.
+
+<b>Today is the day we have been expecting for so long!</b>
+
+We wish everyone an amazing summer (or winter!!) and to our amazing RGSoC 2016 students, coaches, mentors, supervisors and organizers: Let’s get this Summer of Code started! It’s time to code! \o/
```
Comment (anikalindtner) [inline blog/_posts/2016-07-01-RGSoC-2016-day-1.md]: I would switch the \o/ to: 🎉

**Comment by anikalindtner** [comment]
Hey @anasofiapinho I like the blog post! I would implement though a picture of all the team's location again to show how many amazing teams we have and mention the projects they are working on. I also added some tiny comments in the blog post with the goal for us to find a tiny bit more 'serious' tone in our communication with the everyone. Else 👍  from me, it's a wonderful post!

**Comment by alicetragedy** [comment]
Love the post!!!! I really really like how the board and the pictures turned out. Great job @anasofiapinho <3

**Comment by anasofiapinho** [comment]
Thank you so much for your feedback @anikalindtner and @alicetragedy ! <3 It was a tough project and needed some closure. We ended up filling 90% of the board. even though it may have seemed weird that I added the teams to the top of the board, I believe that differentiating them with the rgsoc red squares and the fact that all of them will be giving a lot of their time to the program it made sense to me to have them on the board.

**Comment by anasofiapinho** [comment]
I am ready to do simpler projects now. No arts and crafts please!

**Comment by anasofiapinho** [comment]
I am ready to do simpler projects now. No arts and crafts please! Fyi, no one told me to do arts and crafts! #lessonslearned

**Pull request merged**